### ISSUE TITLE
Add secondary button states

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/design-manual#309]
http://cfpb.github.io/design-manual/ui-toolkit/buttons.html lists the primary button states (hover, active, disabled), but doesn't list the states of the secondary button, beyond the normal state.

**Comment by JenniferHoran** [comment]
This would also be important for 508.

**Comment by mebates** [comment]
Hey @anselmbradford, I think we can assume that the buttons would follow the same pattern as the blue:

Normal: Gray
Hover/focused: Gray 80%
Active: Dark Gray
Disabled: Gray 20% (should match the blue disabled state)

Do you think this needs to be documented in the Design Manual? I think it's pretty easy to discern, but if others don't, we should document it.

**Comment by anselmbradford** [comment]
> Do you think this needs to be documented in the Design Manual? I think it's pretty easy to discern, but if others don't, we should document it.

I'd err on the side of being comprehensive in the design manual and documenting it, particularly since the colors don't use the same name across all states for the primary button. (Someone could get confused by the ambiguity and use Neutral for the Active state, for instance.)

**Comment by mebates** [comment]
@contolini may help out with this!

**Comment by nataliafitzgerald** [comment]
@anselmbradford 
Is this something that's already in Capital Framework? Just clarifying whether this task is just to add it to the Design Manual documentation? 

Is this the code for this? 
<img width="333" alt="screen shot 2017-08-24 at 5 12 07 pm" src="https://user-images.githubusercontent.com/6887128/29688928-6a433016-88ef-11e7-8c81-8485a1611217.png">

**Comment by anselmbradford** [comment]
@nataliafitzgerald yes that's right, see https://cfpb.github.io/capital-framework/components/cf-buttons/#secondary-button
